phpcs.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0"?>
```

Comment:
Because we're distributing it, but others may want to override this locally with their own PHPCS - maybe to test it against the next version of WPCS or some other reason.

You only need to check https://github.com/squizlabs/PHP_CodeSniffer itself to see `.dist` versions of config files, even if the project has it's own standard to use.